api/services/scosService.ts

```diff
@@ -0,0 +1,27 @@
+import request from 'request-promise';
+import getLogger from '../utils/logger';
+
+const { SCOS_HOST } = process.env;
+
+const log = getLogger('scosService');
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+async function makeSCOSRequest(body: any): Promise<any> {
```

Comment:
Why are these any's ok?

Reply:
Those types might be modified in the future, I'm not 100% sure what this will look like. But for now, I wanted this function to support any request to the SCOS API, so the data could really look like anything.